src/uart.rs

```diff
@@ -302,6 +303,116 @@ pub mod config {
         }
     }
 
+    /// Configures the interrupts the UART driver should enable
+    /// in order to be able to quickly inform us about the
+    /// related event.
+    #[derive(Debug, Clone)]
+    pub struct UserInterruptConfig {
+        /// If `Some(number_of_words)`, an interrupt will trigger
+        /// after `number_of_words` could have been transmitted
+        /// (unit is baudrate dependant).
+        ///
+        /// If `None` or `Some(0)` interrupt will be disabled.
+        pub receive_timeout: Option<u8>,
+        /// Sets the threshold **above** which an interrupt will
```

Comment:
The "above" doesn't turn out to be correct, even though I copied it from the official docs.